lib/components/admin/query-record.js

```diff
@@ -13,41 +14,55 @@ import { CallRecordButton, CallRecordIcon } from './styled'
  * Displays information for a query stored for the Call Taker module.
  */
 class QueryRecordLayout extends Component {
-  _getParams = () => planParamsToQuery(parseQueryParams(this.props.query.queryParams))
+  _getParams = () =>
+    planParamsToQuery(parseQueryParams(this.props.query.queryParams))
 
   _viewQuery = () => {
-    const {parseUrlQueryString} = this.props
+    const { parseUrlQueryString } = this.props
     const params = this._getParams()
     parseUrlQueryString(params, '_CALL')
   }
 
-  render () {
-    const {query, timeFormat} = this.props
+  render() {
+    const { query, timeFormat } = this.props
     const params = this._getParams()
-    const time = query.timeStamp
-      ? moment(query.timeStamp).format(timeFormat)
-      : moment(params.time, OTP_API_TIME_FORMAT).format(timeFormat)
+    // Express time in the agency's homeTimezone.
+    //
```

Comment:
`//` can probably go!

Reply:
This was removed in https://github.com/opentripplanner/otp-react-redux/pull/637/commits/0c5291ddc365f9a95f104eb1937916618d47d46f.

Reply:
Ah apologies! Was looking commit by commit to try to drown out the autoformat noise